src/main/java/com/tieumy/exercise/NumeralEx08.java

```diff
@@ -0,0 +1,12 @@
+package com.tieumy.exercise;
+
+public class NumeralEx08 {
+
+    public int convertToBinary(final int n){
+        String binary = Integer.toBinaryString(n);
```

Comment:
Mục đích của bài này là dùng vòng for nên em làm lại nha

Reply:
DONE